web/pdf_page_view.js

```diff
@@ -546,10 +546,9 @@ var PDFPageView = (function PDFPageViewClosure() {
       canvas.height = Math.floor(viewport.height) * PRINT_OUTPUT_SCALE;
 
       // The rendered size of the canvas, relative to the size of canvasWrapper.
-      canvas.style.width = (PRINT_OUTPUT_SCALE * 100) + '%';
+      canvas.style.width = '100%';
 
-      var cssScale = 'scale(' + (1 / PRINT_OUTPUT_SCALE) + ', ' +
-                                (1 / PRINT_OUTPUT_SCALE) + ')';
+      var cssScale = 'scale(1,1)';
```

Comment:
I have tested printing landscape and portrait PDFs and it works as expected (in the browsers print dialog, you have to select the right paper orientation).

I'll inline the scale as you suggested. 

I looked at the other patch and I don't have any comments or feedback at this point. I'll have a closer look a bit later.